custom-options-plus.php

```diff
@@ -347,13 +347,13 @@ function get_custom( $name ) {
 }
 
 // Get your array options
-function get_customs( $name ) {
+function get_customs( $name, $label = false) {
```

Comment:
I'd like to have a [space between parentheses]((https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage).